features/bevragen/persoon/verblijfplaats/onbekend/protocollering-antwoord.feature

```diff
@@ -0,0 +1,34 @@
+#language: nl
+
+Functionaliteit: protocollering van het antwoord voor verblijfplaats onbekend
+
+  Rule: In het antwoord geleverde velden worden geprotocolleerd als de elementnummers volgens Logisch ontwerp BRP
+    Dit is een 6-cijferige code, met zo nodig voorloopnul voor categorieën.
+    Alleen velden die echt geleverd zijn, dus een waarde hebben in de response, worden opgenomen als geleverde rubriek
+    Ook alle velden die worden meegeleverd voor het bepalen van het verblijfplaats type worden opgenomen.
```

Comment:
`Ook alle velden die worden meegeleverd voor het bepalen van het verblijfplaats type worden vastgelegd in het veld 'request_zoek_rubrieken'.`

Reply:
aangepast in alle 4 protocollering-antwoord features